core/templates/services/question-validation.service.spec.ts

```diff
@@ -12,53 +12,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { TestBed } from '@angular/core/testing';
+import { StateEditorService } from 'components/state-editor/state-editor-properties-services/state-editor.service';
+import { QuestionObjectFactory } from 'domain/question/QuestionObjectFactory';
+import { MisconceptionObjectFactory } from 'domain/skill/MisconceptionObjectFactory';
+import { QuestionValidationService } from './question-validation.service';
+
```

Comment:
Move the imports below the file overview. Do it here and in every other file in this PR (if this case occurs). In general, after making your PR, come review the PR yourself. It helps me catch some of my errors and can be very useful to save time in review.

Reply:
Done.